README.md

```diff
@@ -311,25 +311,14 @@ const sendToServer = ({documentCapture, faceCapture}) => {
 
 ## Authentication
 
-Clients are authenticated using JSON Web Tokens (JWTs). The tokens are one use only and expire after 30 minutes. See [here](https://jwt.io/) for details on how JWTs work.
+Clients are authenticated using JSON Web Tokens (JWTs). The tokens expire 90 minutes after being created.
```

Comment:
24 hours at the moment, we will decrease to 90 minutes soon, but this is currently inconsistent with the API docs.